tsec-http4s/src/main/scala/tsec/authentication/ECookieAuthenticator.scala

```diff
@@ -438,7 +444,7 @@ object EncryptedCookieAuthenticator {
         * @return
         */
       def discard(authenticator: AuthEncryptedCookie[A, I]): OptionT[F, AuthEncryptedCookie[A, I]] =
-        OptionT.pure(authenticator.copy[A, I](content = AEADCookie[A]("invalid"), expiry = HttpDate.now))
+        OptionT.pure(authenticator.copy[A, I](content = AEADCookie[A]("invalid"), expiry = Instant.now()))
```

Comment:
`Instant.now` as a side-effect in pure method.

Reply:
Ah, this is true.. I should change this everywhere though. Great catch